argilla/src/argilla/client.py

```diff
@@ -57,9 +51,24 @@ def __init__(
         api_url: Optional[str] = DEFAULT_HTTP_CONFIG.api_url,
         api_key: Optional[str] = DEFAULT_HTTP_CONFIG.api_key,
         timeout: int = DEFAULT_HTTP_CONFIG.timeout,
+        retries: int = DEFAULT_HTTP_CONFIG.retries,
         **http_client_args,
     ) -> None:
-        super().__init__(api_url=api_url, api_key=api_key, timeout=timeout, **http_client_args)
+        """Inits the `Argilla` client.
+
+        Args:
+            api_url: the URL of the Argilla API. If not provided, then the value will try
+                to be set from `ARGILLA_API_URL` environment variable. Defaults to
+                `"http://localhost:6900"`.
+            api_key: the key to be used to authenticate in the Argilla API. If not provided,
+                then the value will try to be set from `ARGILLA_API_KEY` environment variable.
+                Defaults to `None`.
+            timeout: the maximum time in seconds to wait for a request to the Argilla API
+                to be completed before raising an exception. Defaults to `60`.
+            retries: the number of times to retry a failed HTTP request to the Argilla API
+                before raising an exception. Defaults to `5`.
```

Comment:
Based on the[ httpx docs](https://www.python-httpx.org/advanced/transports/), we should change the value description
```suggestion
            retries: the number of times to retry the HTTP connection to the Argilla API
                before raising an exception. Defaults to `5`.
```